# 5' Capping
Adding a methylguanosine to the 5' end of DNA (requires a GTP) soon after the beginning of transcription. Important for splicing the transcript and transport from the nucleus to the cytoplasm. Also serves as the binding site for ribosomes in [[Translation]]. 

**TF-II H** creates a binding site for the cap synthesizing complex. After construction, the cap binding complex associates with RNA Polymerase II and merges it with the RNA.
# 3' Polyadenylation
Adenine sequence added to the 3' end of mRNA, occurring at the termination of mRNA transcription. It protects mRNA from degradation by nucleases and increases translation efficiency.

1. RNA Polymerase II is phosphorylated, creating a binding site for the polyadenylation complex
2. The complex will identify the **cleavage signal (AAUAA)** and will cut it 10-30 nucleotides downstream
3. **Poly A polymerase (PAP)** will synthesize the tail
# Introns
Sequences in RNA which do not code for proteins. Introns allow for **alternative splicing**, or allowing for many different proteins to be made form the same gene. For example for a gene with 5 exons:

**Protein 1**: *Exon1 - Exon2 - Exon3 - Exon5*
**Protein 2**: *Exon2 - Exon3 - Exon4 - Exon5*

**Group I and II introns** are self-splicing and will not be included in RNA automatically. **Group III introns** have to be removed with spliceosomes (require ATP).
## Group I and II splicing mechanism
Both use a nucleophile to to release one the beginning of the intron, allowing following exon to attack the preceding (before the intron) exon strand and pop off the intron entirely. Group I will use GTP, whereas group II will use an -OH on a Uracil.
## Group III splicing mechanism
the mRNA will combine with Small Nuclear Ribonucleoproteins (snRNPs) which are RNA mixed with proteins. This forms a **spliceosome** which contains the snRNPs U1, U2, U4, U5, and U6.

1. U1 and U2 snRNPs bind to intron
2. U4, U5, U6 snRNPs join
3. U1 and U4 snRNPs removed
4. -OH of adenine attacks beginning of intron (proceeds like Groups I/II)
# mRNA
Detecting and degrading abnormal mRNA.

**No-go decay**: Eats mRNA which have secondary structures that block ribosomes.
**Nonsense decay**: Eats mRNA transcripts which contain a premature stop codon upstream of an exon junction complex (marking the beginning/end of an exon).
**Nonstop decay**: Eats mRNA without a stop codon (translating the poly-A tail)